easybuild/easyconfigs/o/OpenMPI/OpenMPI-4.1.1rc2-GCC-10.3.0.eb

```diff
@@ -0,0 +1,35 @@
+##
+# Author:    Robert Mijakovic <robert.mijakovic@lxp.lu>
+##
+name = 'OpenMPI'
+version = '4.1.1rc2'
```

Comment:
So, I'm pretty sure we are definitely not ever going to use a RC in a toolchain, but, this unfortunately means that this blocks the rest of the PR from being merged. It would be better to have had these common GCCcore components split out so one can proceed with the dozens of other stuff we need in GCCcore 10.3

Reply:
Yes, I was following the approach of @boegel, the one he used in GCC 10.3.0. I start with the RC and once the release is there, I will simply replace it with the release. In principle, I have nothing against splitting it into many little pull requests.

Reply:
It's good to stay below 10 new/touched easyconfigs per PR, and use a separate PR for large easyconfigs like `Perl`.

Other than that, your approach of using an RC release is fine.

Reply:
Please bump to `4.1.1` (and fix checksum)